build/charts/antrea-windows/conf/antrea-agent.conf

```diff
@@ -1,7 +1,5 @@
 # FeatureGates is a map of feature names to bools that enable or disable experimental features.
 featureGates:
-# Enable flowexporter which exports polled conntrack connections as IPFIX flow records from each agent to a configured collector.
```

Comment:
@wenyingd @XinShuYang I noticed the feature gate list is almost empty, do you know why we didn't list the support feature gates like we did on linux?

cc @antoninbas